docs/DeveloperGuide.md

```diff
@@ -801,6 +801,22 @@ testers are expected to do more *exploratory* testing.
    
    4. Other incorrect delete commands to try: `deletetask`, `deletetask x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
+
+### Adding a grade to a student
+
+1. Adding a grade to a student while student list is being shown
+
+    1. Prerequisites: List all students using the `liststudent` or `listall` command. There must be at least one student in the list.
```

Comment:
i think for this it would be better if we say `liststudent` only so they can see their grades change